templates/_seo_meta.twig

```diff
@@ -50,6 +50,9 @@
 {% if seomaticSocial.facebookAppId %}
 <meta property="fb:app_id" content="{{ seomaticSocial.facebookAppId |raw }}" />
 {% endif %}
+{% if seomaticSocial.facebookAdminId %}
+<meta property="fb:admins" content="{{ seomaticSocial.facebookAdminId |raw }}" />
```

Comment:
Pedantic: space missing before `raw` on this and line 51 - these should be like this `filter | filter | filter`

Reply:
Ah thanks.  I was copying how it had been done in the existing package 😬